scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -80,19 +80,18 @@
       <thead>
         <tr>
           <th class="w-25">{{ group_name|capitalize }} cases ({{cases|length}}/{{status_ncases[group_name] or 0}})</th>
-          <th class="w-10">Status <a href="" id="{{group_name}}_status" onclick="updateArgs('{{group_name}}','sort=status','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='status' else 'grey'}};background-color:white;">
```

Comment:
I took inspiration from here :https://getbootstrap.com/docs/5.0/utilities/sizing/

Reply:
Bit it's a div

Reply:
Actually, sass doesn't look bad at all: https://sass-lang.com/install, but its again one more thing to add to workflows, and to get to interact with different versions of bootstrap..